wear/src/main/java/io/homeassistant/companion/android/views/ChooseEntityView.kt

```diff
@@ -59,6 +66,31 @@ fun ChooseEntityView(
                     )
                 }
             }
+
+            if (favoriteEntityIds.value.isNotEmpty()) {
+                item {
+                    ExpandableListHeader(
+                        string = stringResource(commonR.string.favorites),
+                        expanded = expandedFavorites,
+                        onExpandChanged = { expandedFavorites = it }
+                    )
+                }
+                if (expandedFavorites) {
+                    items(favoriteEntityIds.value.size) { index ->
+                        val favoriteEntityID = favoriteEntityIds.value[index].split(",")[0]
+                        for (entity in entitiesByDomain.flatMap { (key, values) -> values }) {
+                            if (entity.entityId == favoriteEntityID) {
+                                ChooseEntityChip(
+                                    entity = entity,
+                                    onEntitySelected = onEntitySelected
+                                )
+                                break // no need to search further
+                            }
+                        }
```

Comment:
Using a for loop and breaking it works but using Kotlin language features makes it cleaner in my opinion:

```suggestion
                        entitiesByDomain.flatMap { (_, values) -> values }
                            .firstOrNull { it.entityId == favoriteEntityID }
                            ?.let {
                                ChooseEntityChip(
                                    entity = it,
                                    onEntitySelected = onEntitySelected
                                )
                            }
```

Reply:
Yes, like this style as well 😊
I did this one now, will do the same in `MainView` later today as well!